currency/storage_types.go

```diff
@@ -8,29 +8,19 @@ import (
 	"github.com/thrasher-corp/gocryptotrader/currency/forexprovider"
 )
 
-// CurrencyFileUpdateDelay defines the rate at which the currency.json file is
```

Comment:
Not a huge deal, but I don't know why you moved all of these out of `_types`

Reply:
Just when I was debugging I kept the consts and vars in the same file that they were being used. Happy to revert this change if it's not appropriate.